src/main/java/uk/gov/hmcts/reform/bulkscanning/BulkScanningApiApplication.java

```diff
@@ -14,10 +12,6 @@ public class BulkScanningApiApplication {
 
     public static void main(final String[] args) {
         try{
-            //Setting Liquibase DB Lock property before Spring starts up.
-            LiquibaseConfiguration.getInstance()
-                .getConfiguration(GlobalConfiguration.class)
-                .setUseDbLock(true);
```

Comment:
Deprecated/no longer exists in later versions.

Reply:
Ok, thanks Dan.